schema/release-schema.json

```
@@ -1224,28 +1224,28 @@
         },
         "description": {
           "title": "Budget Source",
-          "description": "A short free-text description of the budget allocation for this contracting process. This may be used to provide human-readable information on the budget category allocated to this contracting process, and/or, information about the nature and source of the allocation (e.g. conditional, confirmed; any official authorizations given to the budget allocation).",
+          "description": "A short free-text description of the budget allocation which will be used in a future contracting process. This may be used to provide human-readable information on the budget category allocated to the future contracts, and/or, information about the nature and source of the allocation (e.g. conditional, confirmed; any official authorizations given to the budget allocation).",
           "type": [
             "string",
             "null"
           ]
         },
         "amount": {
           "title": "Amount",
-          "description": "The value reserved in the budget for this contracting process. A negative value indicates anticipated income to the budget as a result of this contracting process, rather than expenditure. Where the budget is drawn from multiple sources or extends over multiple years, the budget breakdown extension can be used. This field should not be used to report the total value of the budget line funding this contracting process.",
+          "description": "The value reserved in the budget to be used in a future contracting process. A negative value indicates anticipated income to the budget as a result of this contracting process, rather than expenditure. Where the budget is drawn from multiple sources or extends over multiple years, the budget breakdown extension can be used. This field should not be used to report the total value of the budget line funding this contracting process.",
           "$ref": "#/definitions/Value"
         },
         "project": {
           "title": "Project title",
-          "description": "The name of the project through which this contracting process is funded (if applicable). Some organizations maintain a registry of projects, and the data should use the name by which the project is known in that registry.",
+          "description": "The name of the project through which the future contracting process will be funded (if applicable). Some organizations maintain a registry of projects, and the data should use the name by which the project is known in that registry.",
```

Should we do this for `project` and `projectID` to correspond? If so, please also add a changelog entry.

```suggestion
          "description": "The name of the project of which the future contracting process will be part, or through which it will be funded. Some organizations maintain a registry of projects, and the data should use the name by which the project is known in that registry.",
```

---

I was planning to do it later within https://github.com/open-contracting/standard/issues/1517, but we can do it here as well. Done in https://github.com/open-contracting/standard/pull/1513/commits/756b5202b2cea363b576b7618d186e281d01b554.